app/src/resolvers.ts

```diff
@@ -40,7 +41,10 @@ resolver.define('updateJenkinsServer', async (req) => {
 });
 
 resolver.define('getAllJenkinsServers', async (req) => {
-	await adminPermissionCheck(req);
+	if (req.context.moduleKey !== 'jenkins-for-jira-global-page') {
```

Comment:
We only want to run the admin check for the Jira admin folks. From the global page, data can only be viewed as we're hiding all the buttons.